tools/wptrunner/requirements_safari.txt

```diff
@@ -1 +1,2 @@
 mozprocess==1.2.1
+psutil==5.8.0
```

Comment:
cc @jgraham

Reply:
11:08 < jgraham> gsnedders: Hmm, so psutil. We can't vendor that, correct?
11:09 < jgraham> I don't think it's going to break CI for us because it's only used for Safari. So I guess I'm not very concerned, but it would be nice to avoid the dep if we could
11:09 < jgraham> (also I think you can drop all the Python 2 codepaths in that PR)
11:45 < gsnedders> jgraham: I was adding psutil to try and avoid having to implement a whole bunch of low-level stuff with ctypes
11:45 < gsnedders> jgraham: and that was mostly a matter of "do I think it is worth my time to do this"
11:54 < jgraham> gsnedders: Yeah. So I think it's probably fine to include, but there's definitely a chance I'm wrong :)